BeamTutorial/pom.xml

```diff
@@ -43,10 +43,15 @@
                 <filter>
                   <artifact>*:*</artifact>
                   <excludes>
-                    <exclude>META-INF/**</exclude>
+                    <exclude>META-INF/*.SF</exclude>
+                    <exclude>META-INF/*.DSA</exclude>
+                    <exclude>META-INF/*.RSA</exclude>
                   </excludes>
                 </filter>
               </filters>
+	      <transformers>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+	      </transformers>
```

Comment:
This block: is what populates the ServiceLoader configuration and copies it from all the linked-in jars to the bundled jar that `maven-shade-plugin` produces.